search/query_params.hpp

```diff
@@ -115,6 +115,9 @@ class QueryParams
   inline Langs const & GetLangs() const { return m_langs; }
   inline bool LangExists(int8_t lang) const { return m_langs.Contains(lang); }
 
+  inline void SetCategorialRequest(bool rhs) { m_isCategorialRequest = rhs; }
```

Comment:
s/rhs/categorical/

Reply:
Yup, I hesitated here. Changed to isCategorial